FDRS_Gateway/FDRS_Gateway.ino

```diff
@@ -145,13 +145,13 @@ void loop() {
 
   timeClient.update();  //update internal clock if possible
   #endif
-#ifdef USE_SD_LOG
-unsigned long current_millis = millis();
-if(current_millis-last_millis >= 1000){
-  seconds_since_reset+=(current_millis-last_millis)/1000;
-  last_millis=current_millis;
-}
-#endif
+  #ifndef USE_WIFI
```

Comment:
Can you make this also only happen when logging is enabled? Pardon if i'm misreading.

Reply:
Yep, that makes sense!